src/main/java/bio/terra/app/controller/AdminApiController.java

```diff
@@ -61,4 +77,51 @@ public ResponseEntity<JobModel> registerDrsAliases(List<DrsAliasModel> aliases)
     String jobId = drsService.registerDrsAliases(aliases, userReq);
     return ControllerUtils.jobToResponse(jobService.retrieveJob(jobId, userReq));
   }
+
+  @Override
+  public ResponseEntity<DatasetModel> adminRetrieveDataset(UUID id) {
+    List<DatasetRequestAccessIncludeModel> include =
+        List.of(
+            DatasetRequestAccessIncludeModel.ACCESS_INFORMATION,
+            DatasetRequestAccessIncludeModel.DATA_PROJECT,
+            DatasetRequestAccessIncludeModel.PROFILE,
+            DatasetRequestAccessIncludeModel.PROPERTIES,
+            DatasetRequestAccessIncludeModel.SCHEMA,
+            DatasetRequestAccessIncludeModel.STORAGE);
+    AuthenticatedUserRequest userReq = getAuthenticatedInfo();
+    logger.info(
+        String.format(
+            "Verifying resource type admin authorization: %s for resource type: %s and resource id: %s",
+            userReq.getEmail(), IamResourceType.DATASET, id));
```

Comment:
In general you don't want to use `String.format()` with logging since the logger itself can format strings, and depending on log setting some outputs may not be emitted and no formatting needs to occur.
```suggestion
    logger.info(
        "Verifying resource type admin authorization: {} for resource type: {} and resource id: {}",
        userReq.getEmail(), IamResourceType.DATASET, id);
```